.github/workflows/documentation-links.yml

```diff
@@ -0,0 +1,21 @@
+name: Documentation preview of Read the docs in PR
+
+on:
+  pull_request_target:
+    types:
+      - opened
+    paths:
+      - "docs/**"
+      - ".github/workflows/documentation-links.yml"
+
+permissions:
+  pull-requests: write
+
+jobs:
+  documentation-links:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: readthedocs/actions/preview@v1
+        with:
+          project-slug: "viztracer-preview"
```

Comment:
I don't believe this works without the actual readthedocs project.

Reply:
I cannot understand, it can work in other [project](https://github.com/dranjan/python-plyfile/blob/master/.github/workflows/pull-request-doc-preview.yml)

Reply:
Yeah, because https://python-plyfile.readthedocs.io/ exists - you need to register the project on readthedocs first to make this work. There's no `viztracer-preview` project on readthedocs so it won't work. Try it locally and see.

Reply:
You are right, I should learn how it works and I remove the file.